packages/mergeOnGreen/src/index.d.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// define types for a few modules used by probot that do not have their
+// own definitions published. Before taking this step, folks should first
+// check whether type bindings are already published.
+declare module "promise-events" {
+  class EventEmitter {
+  }
+}
```

Comment:
we should update our templates to have a newline at the end of the file (which gets rid of this red mark at the end of the file); I know this drives @JustinBeckwith bonkers :laughing: 

As a goal, I'd like us to update our linters across the whole organization to catch this, so that it's not something that comes up in code review.